src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -218,7 +222,13 @@ internal void BuildAndLogConsoleLog(string msg, LogLevel level)
             }
             else
             {
-                _consoleLogSource?.Log(consoleLog);
+                // These are 'user' console logs and should be forwarded to the ScriptHost, unless
```

Comment:
Not blocking: What makes it obvious that these logs are this stage are just "user" logs? Are we just basing this on the "prefix" that the helpers in WorkerProcessUtilities are checking for? (In this case "LanguageWorkerConsoleLogPrefix" and "ToolingConsoleLogPrefix") ?